### PR TITLE
[connectivity_plus] website docs + example app

### PR DESCRIPTION
**Pull request by pr-Mais**
Part of #113.

This PR includes:
1. Adding the website docs for Connectivity Plus.
2. Small changes to example app.
3. Removing old README docs.

### CODE REVIEW

File: docs/connectivity/overview.mdx [hunk 1]
```diff
@@ -0,0 +1,41 @@
+---
+title: Connectivity Plus Overview
+sidebar_label: Overview
+hide_title: true
+---
+
+## Connectivity Plus Overview
+
+Connectivity plus is Flutter plugin that allows your apps to get information about 
+the network connectivity of the device, wether it's running on cellular or WiFi,
```
Comment (jpnurmi) [inline docs/connectivity/overview.mdx:10]: typo: wether

File: docs/connectivity/usage.mdx [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+#### 1. Declare a StreamSubscription
+
+``` dart
+StreamSubscription<ConnectivityResult> _connectivitySubscription;
+```
+
+#### 2. Listen to the Stream
+
+Use `onConnectivityChanged` method that has a return type `Stream<ConnectivityResult>`, 
+to register a listner.
```
Comment (jpnurmi) [inline docs/connectivity/usage.mdx:57]: typo: listner

File: docs/connectivity/usage.mdx [hunk 3]
```diff
@@ -0,0 +1,125 @@
+---
+title: Connectivity Plus Usage
+sidebar_label: Usage
+hide_title: true
+---
+
+## Using Connectivity Plus
+
+To start using Connectivity Plus, initialize the singelton, which will give you accesss to all methods in the API.
```
Comment (jpnurmi) [inline docs/connectivity/usage.mdx:9]: typo: singelton

**Review by jpnurmi — APPROVED** [review]
A couple of typos spotted but otherwise LGTM :+1:

**Comment by pr-Mais** [comment]
Need to add grammarly to VS code 
Thank you! Will fix and merge

**Pull request merged**